src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -324,4 +324,7 @@
   <data name="ToggleRetroEffectCommandKey" xml:space="preserve">
     <value>Toggle retro terminal effect</value>
   </data>
+  <data name="WriteSettingsFailed" xml:space="preserve">
+    <value>We could not write to your settings file. Check the permissions on that file.</value>

```

Comment:
We might want to be slightly more specific about what "checking the permissions" means like saying

```suggestion
    <value>We could not write to your settings file. Check the permissions on that file to ensure that the read-only flag is not set and that write access is granted.</value>

```